docs/reference/modules/discovery/discovery-settings.asciidoc

```diff
@@ -5,10 +5,13 @@ Discovery and cluster formation are affected by the following settings:
 
 `discovery.seed_hosts`::
 
-    Provides a list of master-eligible nodes in the cluster. Each value has the
-    format `host:port` or `host`, where `port` defaults to the setting
-    `transport.profiles.default.port`. Note that IPv6 hosts must be bracketed.
-    The default value is `["127.0.0.1", "[::1]"]`. See <<unicast.hosts>>.
+    Provides a list of the addresses of the master-eligible nodes in the
+    cluster. Each address has the format `host:port` or `host`, where `port`
+    defaults to the setting `transport.profiles.default.port` falling back to
+    `transport.port` if not set; if neither is set then the default port is
+    `9300`. Note that IPv6 addresses must be bracketed. May also be a single
+    string containing all the addresses separated by commas. The default value
+    is `["127.0.0.1", "[::1]"]`. See <<unicast.hosts>>.
```

Comment:
The new info is great, but the user can't quickly scan to see which settings impact the `port`. I think some formatting and whitespace would help.

```
`discovery.seed_hosts`::
+
--
Provides a list of the addresses of the master-eligible nodes in the cluster.
Each address has the format `host:port` or `host`.

The `port` is determined by checking the following settings in order:

. `transport.profiles.default.port`
. `transport.port`

If none of these settings are specified, the port defaults to `9300`.

Note that IPv6 addresses must be bracketed. May also be a single string
containing all the addresses separated by commas. The default value is
`["127.0.0.1", "[::1]"]`. See <<unicast.hosts>>.
--
```

Reply:
Thanks, yes that's better. I'm never quite sure how to reformat these one-paragraph bits into more richly formatted text, so let's see if my changes worked there...